teuthology/config.py

```diff
@@ -137,7 +137,6 @@ class TeuthologyConfig(YamlConfig):
     objects. Currently it serves as a convenient interface to
     ~/.teuthology.yaml and nothing else.
     """
-    yaml_path = os.path.join(os.path.expanduser('~/.teuthology.yaml'))
```

Comment:
I guess if drop this line you have to update the comment above about "interface to ~/.teuthology.yaml and nothing else".